src/services/currencyHelpers.js

```diff
@@ -0,0 +1,6 @@
+const Currency = require("../models/currency.js")
+
+module.exports.getCurrencyId = async function (currencyName) {
```

Comment:
Why not use arrow function here?

```suggestion
module.exports.getCurrencyId = async currencyName => {
```